apps/prairielearn/src/components/Navbar.tsx

```diff
@@ -234,11 +234,11 @@ function UserDropdownMenu({
   if (
     navbarType === 'student' &&
     course_instance &&
-    (authz_data.authn_has_course_permission_preview ||
-      authz_data.authn_has_course_instance_permission_view)
+    (authz_data?.authn_has_course_permission_preview ||
+      authz_data?.authn_has_course_instance_permission_view)
```

Comment:
When would `authz_data` be expected to be nullish here? Ditto down around line 620.

Reply:
On any error page, `authz_data` would be nullish since there is no early return.

Reply:
Setting it to `authz_data.overrides?.length > 0` here:

```
  } else if (authz_data?.overrides?.length > 0) {
    displayedName = html`${displayedName} <span class="badge text-bg-warning">modified</span>`;
```

would cause a test failure.